core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationLoader.java

```diff
@@ -108,7 +109,16 @@ private static Configuration loadAsConfiguration(Map<String, Object> yamlConfigu
 
         final Configuration configuration = mapToObject(Configuration.class, yamlConfiguration);
         if (strategiesConfiguration != null) {
-            configuration.setStrategiesConfig((Map<String, Object>) strategiesConfiguration);
+            if (strategiesConfiguration instanceof Map) {
+                configuration.setRawStrategyConfigurations((Map<String, Object>) strategiesConfiguration);
+
+            } else if (strategiesConfiguration instanceof List) {
+                HashMap<String, Object> configMap = new HashMap<>();
```

Comment:
`Map<String, String> configMap`

Reply:
no, it's `HashMap<String, Object>` see the parameter in `Configuration` class

Reply:
Sorry I meant `Map<String, Object>`using the interface not the implementation.

Reply:
Ok, yeah, sure :-)